docker/run/Dockerfile

```diff
@@ -50,6 +50,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         # Install general tools required by this Dockefile.
         curl \
+        python3 \
```

Comment:
I think we can and should drop this as it's a dependency of `python3-pip` already (and we don't install Python 2 here either).